live-comms/August-2021-Update.html

```diff
@@ -54,7 +54,7 @@
     <div class="usa-navbar">
       <div class="usa-logo" id="basic-logo">
         <em class="usa-logo__text">
-          <a href="#main-content" title="TDP Project Updates">
+          <a href="/index.html" title="TDP Project Updates">
```

Comment:
@riatzukiza can you confirm that this line of code reflects the change you mentioned in tabletop?